examples/hello-world/step-by-step/cifar10/sag_he/sag_he.ipynb

```diff
@@ -51,34 +51,6 @@
     "! nvflare job create -j /tmp/nvflare/jobs/cifar10_sag_pt_he -w sag_pt_he -sd ../code/fl -force"
```

Comment:
not in this PR, it is complicated to convert now, later if we can pass in additional arguments to ScriptRunner it will be easier. (FOR example HE need WEIGHTS_DIFF but right now we can't pass that)